### PR TITLE
[Bugfix] Update Docker image based on CPU or GPU.

### PR DESCRIPTION
**Pull request by jiyoung-an**
### Summary
- Host machine:
  - OS: Ubuntu 22.04
  - Nvidia Driver: 535.86.05
  - CUDA: 12.2

- Update Docker image based on CPU or GPU.
   - CPU: Used `python:3.10-slim`. Installed TensorFlow via pip3 due to [compatibility issues](https://github.com/tensorflow/tensorflow/issues/52845#issuecomment-1272337911) with local M1 silicon.
   - GPU: Used `tensorflow/tensorflow:2.14.0-gpu`. Since the host machine only requires the NVIDIA® driver (the NVIDIA® CUDA® Toolkit is not required).
- Update Tensorflow2 to version [v2.14.0](https://www.tensorflow.org/install/pip#linux). (NumPy, Keras)
- Reduce Docker image size.

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for handling this!

File: Dockerfile-cpu [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 
-RUN apt-get update
 ENV SHELL /bin/bash
 
-# install npm and python
-RUN apt-get install -y \
+# install apt packages
+RUN apt-get update && \
+    apt-get install -y \
         apt-transport-https
```
Comment (minho-comcom-ai) [inline Dockerfile-cpu]:
```bash
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
````

**Review by minho-comcom-ai — APPROVED** [review]
LGTM

File: Dockerfile-cpu (hunk 1 above)
Comment (jiyoung-an) [reply Dockerfile-cpu]: Done! Thank you for comment 🙂

**Comment by jiyoung-an** [comment]
Thank you for reviews! now merging...

**Pull request merged**